src/qualtrics.ts

```diff
@@ -0,0 +1,40 @@
+import {postMessageToNativeApp} from './post-message';
+
+export const displayQualtricsIntercept = ({
+    interceptId,
+}: {
+    interceptId: string;
+}): Promise<{displayed: true}> =>
+    postMessageToNativeApp({
+        type: 'DISPLAY_QUALTRICS_INTERCEPT',
+        payload: {interceptId},
+    });
+
+type QualtricsProperty<T> = {
+    key: string;
+    value: T;
+};
+
+export const setQualtricsProperties = ({
+    stringProperties,
+    numberProperties,
+    dateTimePropertyKeys,
+}: {
+    stringProperties: Array<QualtricsProperty<string>>;
+    numberProperties: Array<QualtricsProperty<number>>;
+    dateTimePropertyKeys: Array<string>;
```

Comment:
I see we didn't specified that in the spec, but probably is interesting to give these properties a default value (empty array) from js side?

Reply:
lgtm, done